news/_posts/translation/2020-05-12-do-you-speak-prestashop-may-2020.md

```diff
@@ -0,0 +1,131 @@
+
+### Daily language packs updates
+
+Many of you asked why the translated strings were not updated in their shop, even though they have translated and approved strings, so I would like to take this opportunity to explain to you how translation packs are updated in PrestaShop now.
+
+Translations are updated everyday with a 24 hours delay. Every morning our developers send an export request to Crowdin, but since the exporting process takes some time, they download the export made the day before and send the files to [Github](https://github.com/PrestaShop/TranslationFiles). When the repository is updated, a robot will gather the translation files and make them available to download. 
+But one detail to take into account is that only translations of the last version, the 1.7.7, are updated. 
+
+Since PrestaShop 1.7.7 hasn’t been officially released yet, what you can do is to translate and approve strings of the 1.7.7 version, retrieve these strings from translationfiles and upload them on: app/Resources/Translations/{localeCode}. 
+It might be inconvenient for now but PrestaShop's developers team have in the backlog projects to make all new translations available for all shops from the 1.7.5 version and on, so let’s keep a positive mind! :sparkles:
+
+### :two_hearts: Special thanks to newcomers
+
+Many people subscribe to the project(s) of their choice each month, but not so many start translating straightaway. For that reason, the PrestaShop team would like to thank the ones who rushed into the effort immediately! So lots of love to the dedicated: choefer, Çocuk Saatleri (cocuksaatleri), nenowasaleh, Pacho Gonzalez (Pachogzz), Puunaama1 and Stella Georgakopoulou (Stella_Geo). See you soon on Crowdin… and in PrestaShop :stuck_out_tongue_winking_eye:
+
+
+## A few stats
+
+* **37 members** were active on the project this month.
+* A total of **53,384 words have been translated** and **46,675 validated**.
+* All this in **28 different languages**.
+
+Thank you for your involvement!
+
+
+## Top contributors
+
+A lot of you are working every day on Crowdin to have PrestaShop available in many languages, and we can't thank you enough for your dedication! Here are the **most active translators and proofreaders** for May 2020.
+ 
+#### Top 10 translators in May:
+ 
+| |Translator | Language | # Words
+|-|---------- | -------- | ----------------
+ 1. | Rijad Osmanovic (rijado) | Bosnian | 29,098
+ 2. | mirmal | Esperanto | 7,892
+ 3. | cubache | Catalan | 4,058
+ 4. | Çocuk Saatleri (cocuksaatleri) | Turkish | 3,676
+ 5. | Rytis (Mangustux) | Lithuanian | 1,184
+ 6. | Rauno Riikman (weaver) | Finnish | 852
+ 7. | AmirRzd | Persian | 642
+ 8. | Zoran Tejic (zotamal) | Serbian (Latin) | 588
+ 9. | nenowasaleh | Swahili | 568
+10. | Stella Georgakopoulou (Stella_Geo) | Greek | 500
+ 
+ 
+#### Top 10 proofreaders in May:
+ 
+| | Proofreader | Language | # Words
+|-| ---------- | -------- | ----------------
+1. | Rijad Osmanovic (rijado) | Bosnian | 40,220
```

Comment:
Oops, it looks like there is a space missing at the beginning.

Reply:
can you explain where the space is missing ?